doc/basics.md

```diff
@@ -394,6 +396,28 @@ int main(void) {
 }
 ```
 
+The following is a similar example where one first accesses
```

Comment:
I don't see it here, but we should probably include a relevant snippet of twitter.json here. Also, some grammar:

> The following is a similar example where one wants to get the id of the first tweet. To do this, we use `["statuses"].at(0)["id"]`. To break that down:
> 
> 1. Get the list of tweets (the "statuses" key of the document) using `["statuses"]`).
> 2. Get the first tweet using `.at(0)`.
> 3. Get the id of the tweet using `["id"]`.

Reply:
This is good. I am writing something a bit more elaborate in the same style... commit upcoming.